apidef/oas/fixtures_test.go

```diff
@@ -278,7 +280,9 @@ func TestFixtures(t *testing.T) {
 					if tc.Debug || len(tc.Output) == 0 {
 						keys := slices.Sorted(maps.Keys(result))
 
-						t.Log("Changed keys after migration:")
+						if len(keys) > 0 {
+							t.Log("Changed keys after migration:")
```

Comment:
is it changed keys after migration or just keys after mgiration?

Reply:
It's only the differences (flatMap function higher up produces the diff from an empty apidef migration), but depending on if or what you set the behaviour here is a bit flexible. You can enable debug=true and so on.

See /docs/dev/oas-fixtures.md for more detail but consider it flexible; open to simplification